app/bundles/LeadBundle/Translations/en_US/messages.ini

```
@@ -132,6 +132,7 @@ mautic.lead.field.group.extra="Extra"
 mautic.lead.field.group.personal="Personal"
 mautic.lead.field.group.professional="Professional"
 mautic.lead.field.group.social="Social"
+mautic.lead.field.group.no_data="No datas"
```

I have ChatGPTed it 🙂 :

> The word "data" is already plural, so it doesn't need an "s" at the end. Additionally, "data" is often treated as an uncountable noun

```suggestion
mautic.lead.field.group.no_data="No data"
```

---

It was definitely a typo, or no command key when trying to save.